synchronize.yaml

```diff
@@ -10,14 +10,16 @@
     - name: Install ansible on host
       ansible.builtin.pip:
         name: ansible
+        extra_args: "--break-system-packages"
```

Comment:
`With Python 3.11, this snippet will error out, as pip will refuse to install packages alongside the system's. The fix is to tell pip it's OK to "break" your system packages, either using the --break-system-packages parameter, or the PIP_BREAK_SYSTEM_PACKAGES=1 environment variable`

L'explication me parait pas très rassurante, après on est sur la préproduction, on peux tester et voir ce que cela donne.

Reply:
Je n'arrive pas à comprendre pourquoi il y a des version différentes de python entre eclipse et equinoxe et pourquoi il y a ce message. La seule solution que j'ai trouvé pour que la CD passe c'est d'ajouter cet argument

Reply:
Je viens de retester sans cet argument, ça ne passe pas hélas